test.py

```diff
@@ -16,6 +16,25 @@ def test_peers(self):
         self.assertTrue(type(result['peers']) == list)
         self.assertTrue(len(result['peers']) >= 1)
         self.assertEqual(len(result['peers']), result['count'])
+    
+    def test_search(self):
+        result= self.loklak.search('doctor who')
+        self.assertTrue('statuses' in result)
+        self.assertTrue(type(result['statuses']) == list)
+        self.assertTrue(len(result['statuses']) >= 1)
+        self.assertEqual(len(result['statuses']), int(result['search_metadata']['count']))
+
+    def test_status(self):
+        result = self.loklak.status()
+        self.assertTrue('index_sizes' in result)
+        self.assertTrue('client_info' in result)
+
```

Comment:
Also check if the index_sizes also contains
```
"messages"
"mps"
"users"
"queries"
"accounts"
"user"
"followers"
"following"
```
and similarly for `client_info`

Here's a sample response:
```json
{
  "index_sizes" : {
    "messages" : 113411416,
    "mps" : 104,
    "users" : 24804380,
    "queries" : 988,
    "accounts" : 59,
    "user" : 5569347,
    "followers" : 97,
    "following" : 81
  },
  "client_info" : {
    "RemoteHost" : "14.139.85.200",
    "IsLocalhost" : "false",
    "Upgrade-Insecure-Requests" : "1",
    "Accept-Language" : "en-US,en;q=0.8",
    "If-Modified-Since" : "Sun, 06 Dec 2015 12:47:13 GMT",
    "Host" : "loklak.org",
    "Accept-Encoding" : "gzip, deflate, sdch",
    "X-Forwarded-For" : "172.30.106.127, 14.139.85.200",
    "X-Real-IP" : "14.139.85.200",
    "Via" : "1.1 proxy14.nitw (squid/3.1.8)",
    "User-Agent" : "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.73 Safari/537.36",
    "Accept" : "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
    "Connection" : "close",
    "Cache-Control" : "max-age=259200"
  }
}
```